docs/sources/write/shortcodes/index.md

```diff
@@ -154,6 +154,62 @@ This shortcode inserts a lists of links to the page's subpages and includes the
 {{</* section withDescriptions="true"*/>}}
 ```
 
+## `docs/reference` shortcode
+
+The `docs/reference` shortcode offers more flexible linking than the Hugo built-in `relref` shortcode.
+
+Use this code when content from one repository is published to more than one documentation set, because it lets you specify appropriate links for each doc set in one part of the file (usually at end of the file, like a footer) while using the link label in the body text.
+
+For example, a page in versioned Grafana documentation is also mounted in the Grafana Cloud documentation.
+The page in Grafana should link to the Grafana dashboards page but the page in Grafana Cloud should link to the Grafana Cloud dashboards page.
+
+Set the reference at the end of the document as follows:
+
+```markdown
+{{%/* docs/reference */%}}
+[dashboards]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/dashboards"
+[dashboards]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/dashboards"
+{{%/* /docs/reference */%}}
+```
+
+The content within the shortcode tags is as follows:
+
+- `label` - The label for the shortcode. This is the text that you'll use in the body text. In the example above, the label is `dashboards`. The label can be multiple words (for example, [dashboard docs]) and can include spaces.
```

Comment:
Suggestion from @JStickler on closed #271. I like it, but want to make sure you're good with it @jdbaldry. 

```suggestion
- `label` - The name for this shortcode. This is the text you will use when you insert this specific `doc/reference` in your content. In the example above, the label is `dashboards`. The label can be multiple words (for example, [dashboard docs]) and can include spaces.
```

Reply:
I think "The name for this shortcode" confuses some terminology. The name of the shortcode is `docs/reference`. The label is a label for the link.

Reply:
@JStickler and @jdbaldry - what about (note, I've used the contraction of "you will" per our style guide):
```suggestion
- `label` - The link label for this shortcode. This is the text you'll use when you insert this specific `doc/reference` in your content. In the example above, the label is `dashboards`. The label can be multiple words (for example, [dashboard docs]) and can include spaces.
```

Reply:
I'm happy with that. It's technically accurate.

Reply:
Yeah, my suggestion wasn't 100% clear that that I meant that "label" = "text you use to identify this **instance** of the shortcode".  Maybe insert the word "instance" into your revision Isabel?

Reply:
That *might* not work, but I could be wrong. Could you have more than one label set up in your reference @jdbaldry? I think so, based on other examples we looked at. It probably wouldn't be our use case, but I just want to make sure I'm not adding language that could be a problem later.

Reply:
Sorry, I think I misread your suggestion the first time around.


```suggestion
- `label` - A label associated with one or more reference-style links in the file. 
  In the example above, the label is `dashboards`. The label can be multiple words (for example, `[dashboard docs]`) and can include spaces.`
```

Reply:
I think this language is less clear and direct and using "a" instead of "the" makes it less consistent with the other two descriptions. Suggest:
```suggestion
- `label` - The label you'll use in the reference-style links in the file. In the example above, the label is `dashboards`. The label can be multiple words (for example, [dashboard docs]) and can include spaces.
```

Reply:
LGTM

Reply:
LGTM too